app/controllers/catalog_controller.rb

```diff
@@ -67,6 +67,26 @@ class CatalogController < ApplicationController
     config.add_facet_field 'dc_rights',   field: 'cho_dc_rights_ssim', limit: true
     config.add_facet_field 'provided_by', field: 'agg_data_provider_ssim', limit: true
 
+    # "administrative"-like facets
```

Comment:
see #238. Do we want these to be `if: :superadmin?` too?

Reply:
Eventually? I figured we could start here and see if they're useful first.